src/routes/appointments.js

```diff
@@ -0,0 +1,12 @@
+const router = require('express').Router();
+const Joi = require('joi');
+const validator = require('express-joi-validation').createValidator({});
+const { getByDate } = require('../controllers/appointments');
+const wrapper = require('../middleware/wrapper');
+
+const validatorGetByDate = Joi.object({
+	getByDate: Joi.date().required(),
+});
+router.get('/:getByDate', validator.params(validatorGetByDate), wrapper(getByDate));
```

Comment:
Lets create validation middleware called `validationParams`  to wrap validatorGetByDate instead of using `validator.params`

Reply:
Good point about creating middleware for it, but I'm not sure about middleware just for params, because it's so many unnecessary middlewares, maybe would be a good way to create single middleware for validator and use its methods. I don't know, it's needed to check, I'll find some way.